### DISCUSSION TITLE
What to use as repo for local registry

### DISCUSSION THREAD

**Discussion by develar** [werf/werf#3630] [General]
I have running local registry `localhost:5005`.
Added it as `insecure-registries` to docker.
Checked that it works "curl http://localhost:5005/v2/_catalog ".

But still on `werf converge --dev --repo=localhost:5005 --insecure-registry=true`

I get

Error: unable to get synchronization client id: unable to get repo localhost:5005 tags: parsing repo "localhost:5005": repository can only contain the runes `abcdefghijklmnopqrstuvwxyz0123456789_-./`: localhost:5005

In quckstart "registry.example.com:80" is used, but it is unclear, to what should I replace "registry.example.com".

**Comment by develar** [comment]
Found: `127.0.0.1:5000/yourUserName/yourAppName`

`werf converge --dev --repo=127.0.0.1:5000/develar/cool-app --insecure-registry=true`

**Reply by alexey-igrychev** [reply]
When you create and configure it, it is about **the registry** (`127.0.0.1:5000`, `registry.hub.docker.com`, `ghcr.io`, ...), and when you use it, you need **a certain repository** (`127.0.0.1:5000/<project>`, `registry.hub.docker.com/username/project`, ...) in the registry.
Thank you for your feedback.
